messages/agent.create.md

```diff
@@ -1,25 +1,25 @@
 # summary
 
-Create an Agent from an agent spec.
+Create an agent in your org from a local agent spec file.
 
 # description
 
-Create an Agent from an agent spec. Agent metadata is created in the target org and retrieved to the local project.
+To generate an agent spec file, run the "agent generate spec" CLI command, which outputs a JSON file with the list of jobs and descriptions that the new agent can perform. Then specify this generated spec file to the --job-spec flag of this command, along with the name of the new agent.
 
-# flags.job-spec.summary
+When this command finishes, your org contains the new agent, which you can then edit in the Agent Builder UI. The new agent already has a list of topics and actions that were automatically created from the list of jobs in the provided agent spec file. This command also retrieves all the metadata files associated with the new agent to your local DX project.
```

Comment:
This is true with the current implementation but not necessarily with the v2 implementation.

Reply:
Ditto